.github/workflows/release.yaml

```diff
@@ -5,50 +5,30 @@ on:
     # Sequence of patterns matched against refs/tags
     tags:        
       - "v*.*.*"
-
 jobs:
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: ./tools/github-actions/setup-deps
-
-      # lint
-      - run: make lint-deps
-      - run: make -k lint
-
   release:
     runs-on: ubuntu-latest
-    needs: [ lint ]
     steps:
       - uses: actions/checkout@v3
-      - uses: ./tools/github-actions/setup-deps
 
-      - name: Extract Tag value
-        id: tag_env
+      - name: Extract Release Tag and Commit SHA
+        id: vars
         shell: bash
         run: |
-          echo "##[set-output name=version;]$(echo ${GITHUB_REF##*/})"
-
-      # test
-      - run: make go.test.unit
+          echo "::set-output name=release_tag::$(echo ${GITHUB_REF##*/})"
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
 
-      # build and push image
       - name: Login to DockerHub
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
-
-      - name: Build and Push EG Image
-        env:
-          EG_TAG_VERSION: ${{ steps.tag_env.outputs.version}}
-        run: make image.push.multiarch TAG=${EG_TAG_VERSION} IMAGE_NAME=gateway PLATFORMS="linux_amd64 linux_arm64"
+      - name: Retag and push existing gateway-dev image
+        run: |
+          skopeo copy --all docker://docker.io/envoyproxy/gateway-dev:${{ steps.vars.outputs.sha_short }} docker://docker.io/envoyproxy/gateway:${{ steps.vars.outputs.release_tag }}
```

Comment:
I remembered that we are using envoyproxy/gateway-dev:latest as the dev image, but not building image with commit as the image tag. Is this way working as expectation, I guess it can not fetch image with commitID.

Reply:
How about when building a dev image, we not only build it with tag `latest`, but also tag it with the commit id, using `git rev-parse --short HEAD`

Reply:
it pushes both https://hub.docker.com/r/envoyproxy/gateway-dev/tags

Reply:
<img width="1267" alt="Screen Shot 2022-09-01 at 12 28 22 PM" src="https://user-images.githubusercontent.com/9469142/187996770-f4533c35-bae0-444d-b0eb-a7b012410940.png">

the skopeo cmd looks good, its failing because that SHA doesn't exist yet :)

Reply:
Nope, I think dev workflow does not push image with commit id now, need to add this first. @arkodg